addons/binding/org.openhab.binding.energenie/SSL/mihome.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Source: https://github.com/openhab/openhab2-addons/pull/2054#discussion_r108501988

I do want to add this discussion here:

Martinvw:
> What is this certificate can/will it expire? what then?

svilenvul  (30th of march)
> Yes, it will expire in six months.

... I guess the .crt file should be replaced..

I now it isn't easy for maintenance, I could try to download the certificate at bundle start..